### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by apanicker-nflx** [Netflix/conductor#2180] [Announcements]
## What’s Changed
Please refer the [Changelog ](https://github.com/Netflix/conductor/blob/main/CHANGELOG.md) for a detailed list of changes as part of this release.

<hr /><em>This discussion was created from the release <a href='https://github.com/Netflix/conductor/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by maheshyaddanapudi** [comment]
What will be the new features added in 3.X ?

It's now Spring Boot based which gives a lot of flexibility to add features like Spring Security or LogBook (to log req / res json for better log aggregation) etc.

**Reply by apanicker-nflx** [reply]
We are currently compiling the list of features and enhancements to be made in 3.x We will be publishing the roadmap and feature set over the next couple of weeks once this is finalized. Thank you.

**Reply by apanicker-nflx** [reply]
We have published the roadmap with enhancements and proposed new features [here](https://github.com/Netflix/conductor/wiki/Roadmap). This is not an exhaustive list and please free to propose new features that you may be interested in and would be beneficial to the wider community.
